src/main/java/build/buildfarm/server/BuildFarmServer.java

```diff
@@ -1,3 +1,5 @@
+// “Copyright 2021 VMware, Inc.”
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Remove additional copyright header.

Reply:
Yeah, strong negative here.

Reply:
Our employer requires us to retain the copyright for code added (apparently even small diffs to existing files).

Would it be acceptable to add the employer (VMware) to the AUTHORS file? Similar to https://github.com/bazelbuild/bazel-buildfarm/pull/844?

Reply:
I think that would acceptable, especially since VMware will be a regular contributor to the project.

Reply:
Removed the copyright stuff and added `vmware` in authors file.